doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,519 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "722250c2-6461-4934-ab19-e8ef3cf3b2e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "df.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    fontsize={'title': '16pt', 'xlabel': '12pt', 'ylabel': '14pt', 'ticks': 10},\n",
+    "    title=\"Penguins Measurements by Species\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b7b0baa-bcc9-4195-b721-84dca8006cbe",
+   "metadata": {},
+   "source": [
+    "(option-rescale_discrete_levels)=\n",
+    "## `rescale_discrete_levels`\n",
+    "\n",
+    "The `rescale_discrete_levels` option improves the visual contrast of discrete values when using `cnorm='eq_hist'`. By default, it adjusts the lower bound of the colormap so that non-zero values appear higher on the scale–helpful when low counts would otherwise appear faded.\n",
+    "\n",
+    "This only has an effect if:\n",
+    "- `cnorm='eq_hist'` is set\n",
+    "- The color values are discrete (e.g., counts or categories)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8f08a504-dde4-435c-acf2-2e77449b402c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "\n",
+    "# Simulate discrete values by binning magnitude\n",
+    "df['mag_bin'] = df['mag'].round()\n",
+    "\n",
+    "df.hvplot.scatter(\n",
+    "    x='lon',\n",
+    "    y='lat',\n",
+    "    c='mag_bin',\n",
+    "    cmap='viridis_r',\n",
+    "    cnorm='eq_hist',\n",
+    "    rescale_discrete_levels=True,\n",
+    "    title=\"Earthquakes with Histogram-Equalized Discrete Color Scaling\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "353f7840-ab51-4ba5-a89c-78415c43fca3",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "(option-robust)=\n",
+    "## `robust`\n",
+    "\n",
+    "The `robust` option adjusts how the colormap range is computed for image plots. When set to `True` and no explicit color limits (`clim`) are provided, hvPlot calculates the color limits based on the 2nd and 98th percentiles rather than the extreme minimum and maximum values, reducing the impact of outliers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40221654-a338-4533-9a5a-6cd29e28a94d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvsampledata\n",
+    "import hvplot.xarray  # noqa\n",
+    "\n",
+    "ds = hvsampledata.air_temperature(\"xarray\").sel(time=\"2014-02-25 12:00\")\n",
+    "\n",
+    "plot1 = ds.hvplot.image(width=350)\n",
+    "plot2 = ds.hvplot.image(robust=True, width=350)\n",
+    "\n",
+    "plot1 + plot2"
```

Comment:
Let's use this example from xarray, it makes it more obvious `robust` can help with outliers https://docs.xarray.dev/en/latest/user-guide/plotting.html#robust